src/controller/modules_fn.py

```diff
@@ -135,11 +150,19 @@ def multiplyImages(widget):
     """
     compute multiplication of all input images
     """
-    parent_names = getParentNames(widget)
-    for parent_name in parent_names:
-        if parent_name not in _IMAGES_STACK.keys():
-            return print("'{}' not in image stack".format(parent_name))
-    im = core.multiplyImages([_IMAGES_STACK[parent_name] for parent_name in parent_names])
+    if widget.singleValue.isChecked():
+        try:
+            value = float(widget.value.text())
+        except ValueError:
+            return print("multiply value ERROR")
+        ref_parent_name = widget.reference.currentText()
+        im = core.divideImages(_IMAGES_STACK[ref_parent_name], value=value)
```

Comment:
Same error, we are in multiplication no?

Reply:
RE My mistake